localstack/services/lambda_/invocation/execution_environment.py

```diff
@@ -155,8 +155,9 @@ def get_environment_variables(self) -> Dict[str, str]:
             env_vars.update(self.function_version.config.environment)
         if config.LAMBDA_INIT_DEBUG:
             # Disable dropping privileges because it breaks debugging
-            env_vars["LOCALSTACK_USER"] = ""
-        if config.LAMBDA_INIT_USER:
+            env_vars["LOCALSTACK_USER"] = "root"
+        # Forcefully overwrite the user might break debugging!
+        if config.LAMBDA_INIT_USER is not None:
```

Comment:
@LS devs: Just be careful when using `LAMBDA_INIT_USER` in combination with init debugging.

We could also consider flipping this 🤷